browser/ui/android/strings/android_brave_strings.grd

```diff
@@ -1420,12 +1420,6 @@ Are you sure you want to do this?
       <message name="IDS_RETRY" desc="Retry button text.">
         Retry
       </message>
-      <message name="IDS_PRIVACY_STATS" desc="Private stats title text.">
```

Comment:
references to these were removed here https://github.com/brave/brave-core/pull/12743/files#diff-4326ff8a702dc0509f107bbfac92c3814a71d1e6bc8f43cc50f626adb45fb216